ksqldb-test-util/src/main/resources/log4j.properties

```diff
@@ -18,6 +18,9 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
+# help debug a flaky test
+log4j.logger.io.confluent.ksql.api.server=DEBUG
```

Comment:
this only applies to the tests

Reply:
This is temporary or permanent? If it's the latter does it add much to test tun time?

Reply:
It can be either or - it appears not to add any time to the total PR build. You can compare https://jenkins.confluent.io/job/confluentinc-pr/job/ksql/job/PR-8558/2/ (this one, which took 2h21m) with https://jenkins.confluent.io/job/Confluentinc%20Contributors/job/ksql/job/PR-8549/6/ (#8549 - the most recently merged PR with green on Jenkins, which took exactly 2h21m as well)

Reply:
Cool, thanks!